src/server/types.ts

```diff
@@ -6,8 +6,13 @@ import { InnerRenderFunction, RenderContext } from "./render.tsx";
 
 export type StartOptions = ServeInit & FreshOptions;
 
+export interface RouterOptions {
+  trailingSlash?: boolean;
```

Comment:
Can you please add the `@default` JSDoc directive and briefly document the purpose of this option?